src/pybind/mgr/dashboard/base.html

```diff
@@ -413,6 +413,13 @@
             <li class="ceph-none-found" rv-hide="filesystems | length">None found</li>
           </ul>
         </li>
+
+        <li class="{%if path_info.startswith('/rgw')%}active{%endif%}">          
+            <a href="{{ url_prefix }}/rgw">                                                        
+ <i class="fa fa-files-o" aria-hidden="true"></i>
+            <span>RGW</span></a>                                                           
```

Comment:
Isn't the "official" name "Object Store" or "Object Gateway"? At least that's what the documentation refers to nowadays (albeit not consistently).

Reply:
The docs do have object gateway.  I think that's pretty formal, and RGW is the shortform everywhere.  Just offhand, what do you prefer?

Reply:
We've settled to using "Object Gateway" in our project - we found "RGW" or "RADOS Gateway" to be too non-descriptive in a Web UI.